docs/conf/australia/2022/cfp-extend.rst

```diff
@@ -0,0 +1,28 @@
+:template: {{year}}/generic.html
```

Comment:
(applies globally) I'd probably avoid copying over any old blog posts so that we don't accidentally push out duplicate/outdated content. 

Also, generally we try to keep all blog post in conf/city/year/news directory. Not sure how these ended up in the root year folder, but once we clean this up we should start with a blank slate :)